### ISSUE TITLE
Disable/enable rubocop linters via comments

### ISSUE THREAD

**Issue by thehappycoder** [sds/slim-lint#29]
Could you give me a hint on why the following does not work?

I still get [W] LineLength: Line is too long. [461/120]

```
/ rubocop:disable Metrics/LineLength
        iframe [class="embed-responsive-item"
          src="https://www.productreview.com.au/w/reviews/280683.html?_sf_ignore_cache=1&amp;api_version=2&amp;referer=http%3A%2F%2Fwww.carnextdoor.com.au%2Fcar-next-door-reviews%2F&amp;url=https://www.productreview.com.au/w/reviews/&amp;css_override=false&amp;min_review_rating=0&amp;min_avg_rating=0&amp;order=best&amp;num_reviews=3&amp;width=800&amp;height=290&amp;preview=true&amp;xdm_e=http%3A%2F%2Fwww.carnextdoor.com.au&amp;xdm_c=default1590&amp;xdm_p=1"
          name="easyXDM_default1590_provider" id="easyXDM_default1590_provider"]
        / rubocop:enable Metrics/LineLength
```

**Comment by sds** [comment]
You can't enable/disable RuboCop cops in Slim templates unless you are writing Ruby code like so:

```slim
-# rubocop:disable ...
- some_code ...
-# rubocop:enable ...
```

In this case, it looks like you're trying to disable the `LineLength` linter of SlimLint, not RuboCop. There's currently an open issue tracking this in #12.

**Comment by jhubert** [comment]
fwiw, `-# rubocop:disable ...` seems to trigger:

```
...html.slim:24 [W] CommentControlStatement: Slim code comments (`/ rubocop:disable Style/IdenticalConditionalBranches`) are preferred over control statement comments (`-# rubocop:disable Style/IdenticalConditionalBranches`)
```

but, using the recommended code comment style doesn't disable the cop. 🐑

**Comment by Sharparam** [comment]
Any further information regarding the previous comment, @sds? I came across this issue today as well.

**Comment by sds** [comment]
Thanks for mentioning, @Sharparam. This is fixed in [0.17.1](https://github.com/sds/slim-lint/releases/tag/v0.17.1)
